app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -16,8 +16,8 @@
 @include foundation-close-button;
 @include foundation-drilldown-menu;
 @include foundation-dropdown;
-@include foundation-dropdown-menu;
 @include foundation-menu;
+@include foundation-dropdown-menu;
```

Comment:
can you put a comment in here that this needs to go below the menu, so we don't forget in 6 months :)